databricks/koalas/frame.py

```diff
@@ -6773,6 +6799,27 @@ def __class_getitem__(cls, params):
         # Koalas DataFrame.
         is_dataframe = None
 
+    def _get_dropped_internal_frame(self, columns):
```

Comment:
Can you fix drop to remove fields from Spark DataFrame first and reuse `drop` inside of `pop` API?

Reply:
@HyukjinKwon got it, i've created related PR at #794

Reply:
@HyukjinKwon Fixed it :)